samples/apps/copilot-chat-app/webapp/src/libs/services/SpeechService.ts

```diff
@@ -1,23 +1,17 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import * as speechSdk from 'microsoft-cognitiveservices-speech-sdk';
+import { BaseService } from './BaseService';
 
 interface TokenResponse {
     token: string;
     region: string;
     isSuccess: boolean;
 }
 
-interface SpeechServiceRequest {
-    commandPath: string;
-    method?: string;
-}
-
-export class SpeechService {
-    constructor(private readonly serviceUrl: string) {}
-
-    validSpeechKeyAsync = async () => {
-        const response = await this.invokeTokenAsync();
+export class SpeechService extends BaseService {
+    validateSpeechKeyAsync = async (accessToken: string) => {
```

Comment:
What's the point of this method if there's no validation in it? Couldn't you just call invokeTokenAsync?